src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -0,0 +1,1571 @@
+
+			SetupOutputVariable( "Cooling Panel Total Cooling Energy [J]", CoolingPanel( CoolingPanelNum ).Energy, "System", "Sum", CoolingPanel( CoolingPanelNum ).EquipID, _, "ENERGYTRANSFER", "COOLINGPANEL", _, "System" );
+			SetupOutputVariable( "Cooling Panel Total System Cooling Energy [J]", CoolingPanel( CoolingPanelNum ).TotEnergy, "System", "Sum", CoolingPanel( CoolingPanelNum ).EquipID, _, "ENERGYTRANSFER", "COOLINGPANEL", _, "System" );
+			SetupOutputVariable( "Cooling Panel Convective Cooling Energy [J]", CoolingPanel( CoolingPanelNum ).ConvEnergy, "System", "Sum", CoolingPanel( CoolingPanelNum ).EquipID );
+			SetupOutputVariable( "Cooling Panel Radiant Cooling Energy [J]", CoolingPanel( CoolingPanelNum ).RadEnergy, "System", "Sum", CoolingPanel( CoolingPanelNum ).EquipID );
+
+			SetupOutputVariable( "Cooling Panel Water Mass Flow Rate [kg/s]", CoolingPanel( CoolingPanelNum ).WaterMassFlowRate, "System", "Average", CoolingPanel( CoolingPanelNum ).EquipID );
+			SetupOutputVariable( "Cooling Panel Water Inlet Temperature [C]", CoolingPanel( CoolingPanelNum ).WaterInletTemp, "System", "Average", CoolingPanel( CoolingPanelNum ).EquipID );
+			SetupOutputVariable( "Cooling Panel Water Outlet Temperature [C]", CoolingPanel( CoolingPanelNum ).WaterOutletTemp, "System", "Average", CoolingPanel( CoolingPanelNum ).EquipID );
+		}
+
+	}
+
+	void
+	InitCoolingPanel(
+		int const CoolingPanelNum,
+		int const ControlledZoneNumSub,
+		bool const FirstHVACIteration
+	)
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Russ Taylor
+		//                      Rick Strand
+		//       DATE WRITTEN   Nov 1997
+		//                      Feb 2001
+		//       MODIFIED       Aug 2007 Daeho Kang (Add radiant component)
+		//                      Sept 2010 Brent Griffith (plant interactions)
+		//						Sept 2014 Rick Strand (modified from version in baseboard unit for the cooling panel)
+		//       RE-ENGINEERED  na
+
+		// PURPOSE OF THIS SUBROUTINE:
+		// This subroutine initializes the cooling panel units, and determines the UA values during simulation.
+
+		// METHODOLOGY EMPLOYED:
+		// The initialization subrotines borrowed from other sources and heat exchanger formulation for cooling panel.
+
+		// REFERENCES:
+		// 1. Incropera and DeWitt, Fundamentals of Heat and Mass Transfer
+
+		// Using/Aliasing
+		using DataGlobals::BeginEnvrnFlag;
+		using DataGlobals::NumOfZones;
+		using DataLoopNode::Node;
+		using PlantUtilities::InitComponentNodes;
+		using DataPlant::ScanPlantLoopsForObject;
+
+		// Locals
+		// SUBROUTINE ARGUMENT DEFINITIONS:
+
+		// SUBROUTINE PARAMETER DEFINITIONS:
+		static std::string const RoutineName( "ChilledCeilingPanelSimple:InitCoolingPanel" );
+
+		// INTERFACE BLOCK SPECIFICATIONS
+		// na
+
+		// DERIVED TYPE DEFINITIONS
+		// na
+
+		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+		static bool MyOneTimeFlag( true );
```

Comment:
This should be move up to the top and renamed so it can be reset in a clear_state function.  Same for the getinput flag.